3-box-model/style.css

```
@@ -0,0 +1,29 @@
+footer {
+    background-color: rgb(0, 0, 0);
+    border: 3px  orangered solid;
+    bottom:0;
+    color: white;
+    position: fixed;
+    width: 100%;
+}
+header {
+    background-color: rgb(0, 0, 0);
+    border: 3px solid greenyellow;
+    color: white;
+    position: sticky;
+    width: 100%;
+}
+nav{
```

```suggestion
nav {
```
Be consistent, always add spaces before brackets ;)